src/Riok.Mapperly.Abstractions/MapperAttribute.cs

```diff
@@ -6,6 +6,11 @@ namespace Riok.Mapperly.Abstractions;
 [AttributeUsage(AttributeTargets.Interface | AttributeTargets.Class)]
 public sealed class MapperAttribute : Attribute
 {
+    /// <summary>
+    /// The default property name mapping strategy.
```

Comment:
Is it the default? It is the only property name mapping strategy one can set, isn't it?
I'd probably rephrase to `Strategy on how to match mapping property names`.

Reply:
yeah it isnt default when this is the only place you can set it, will change that.